kernel/src/main/java/org/kframework/parser/ParserUtils.java

```diff
@@ -155,8 +155,15 @@ else if (di instanceof Require) {
 
                 String definitionFileName = ((Require) di).getValue();
 
+                if (definitionFileName.equals("ffi.k")) {
+                    kem.registerCompilerWarning(ExceptionType.FUTURE_ERROR,
+                        "Requiring a K file in the K builtin directory via " +
+                        "a deprecated filename. Please replace \"" + definitionFileName +
+                        "\" with \"" + definitionFileName.substring(0, definitionFileName.length() - 2) + ".md\".", di);
+                }
+
                 ArrayList<File> allLookupDirectoris = new ArrayList<>(lookupDirectories);
-                allLookupDirectoris.add(0, currentDirectory);
+                allLookupDirectoris.add(1, currentDirectory);
```

Comment:
1.  This looks like a typo in the name of this variable `allLookupDirectoris => allLookupDirectories`
2.  What is this change? The `0 => 1`.

Reply:
@dwightguth why was this changed from 0 to 1?
Someone on discord complained about kompile not finding their file: https://github.com/runtimeverification/k/issues/3695